docs/user/operation-guides/02-30-enable-istio-access-logs.md

```diff
@@ -3,18 +3,18 @@ title: Enable Istio Access Logs
 ---
 
 You can enable [Istio access logs](https://istio.io/latest/docs/tasks/observability/logs/access-log/) to provide fine-grained details about the access to workloads that are part of the Istio service mesh. This can help indicate the four “golden signals” of monitoring (latency, traffic, errors, and saturation) and troubleshooting anomalies.
-The Istio setup shipped with Kyma provides a pre-configured [extension provider](https://istio.io/latest/docs/tasks/observability/telemetry) for access logs, which will configure the istio-proxies to print access logs to stdout using JSON format. It uses a configuration similar to the following one:
-
-```yaml
-  extensionProviders:
-    - name: stdout-json
-      envoyFileAccessLog:
-        path: "/dev/stdout"
-        logFormat:
-          labels:
-            ...
-            traceparent: "%REQ(TRACEPARENT)%"
-            tracestate: "%REQ(TRACESTATE)%"
+The Istio setup shipped with the Istio module provides a pre-configured [extension provider](https://istio.io/latest/docs/tasks/observability/telemetry) for access logs, which configures the Istio proxies to print access logs to stdout using the JSON format. It uses a configuration similar to the following one:
+
+```bash
```

Comment:
The format of this is structure and the [file](https://github.com/kyma-project/istio/blob/494f07ad9fe487ca65c5ee83f464d8a83e1c885b/internal/istiooperator/istio-operator.yaml#L153) where this configured is yaml. Why do we change it to bash?